### ISSUE TITLE
tf.train.Saver in facenet_train_classifier.py

### ISSUE THREAD

**Issue by dahangli** [davidsandberg/facenet#267]
I see somewhere that :<train_softmax.py> 
                 saver = tf.train.Saver(tf.trainable_variables(), max_to_keep=3)
==》        saver = tf.train.Saver(tf.global_variables(), max_to_keep=3)

I found "validate_on_lfw.py" and “train_softmax.py“ with "--lfw_dir" won't work if didn't change that.
   
 Should I？

**Comment by davidsandberg** [comment]
Hi,
By saving global variables the saved model becomes significantly larger than when saving trainable variables. I know that some trained parameters in the batch norm layer has not been in the collection of trainable variables, but that should have been fixed for the models that uses slim.

**Comment by dahangli** [comment]
I found that tensorflow 1.1 will work well , and tensorflow 1.0 need changing to save global_variables
